cdc/sinkv2/ddlsink/mq/ddlproducer/kafka_ddl_producer.go

```diff
@@ -67,12 +67,14 @@ func NewKafkaDDLProducer(ctx context.Context, client sarama.Client,
 		go func() {
 			err := client.Close()
 			if err != nil {
-				log.Error("Close sarama client with error", zap.Error(err),
+				log.Error("Close sarama client with error in kafka "+
```

Comment:
The width is less than 80 and why do we need split the `Close sarama client with error in kafka DDL producer` into two lines?